### ISSUE TITLE
Checkbox component doesn't support required

### ISSUE THREAD

**Issue by Blackbaud-AlexKingman** [blackbaud/skyux-forms#76]
### Expected behavior
Consumer should be able to add a `required` and `aria-required` attributes to the `<input type="checkbox" />` element.

### Actual behavior
There is no way to add these attributes.

Notes:
- Should the component detect required state and add the `sky-control-label-required` class as well (it it is not already defined)? This will show the red * to visually indicate the required state.
- This can be automatic, by leveraging native controls. For example, template-driven forms have the `required` attribute; reactive forms have the required validator.
